mkdocs.yml

```diff
@@ -72,11 +74,16 @@ nav:
               - user_guides/fs/feature_group/index.md
               - Create: user_guides/fs/feature_group/create.md
               - Create External: user_guides/fs/feature_group/create_external.md
-              - Deprecating: user_guides/fs/feature_group/deprecation.md
+              - Deprecate: user_guides/fs/feature_group/deprecation.md
```

Comment:
Maybe `Deprecated` instead?

Reply:
I would leave it as `Deprecate`, similar to `Create`.
It is a guide on how to deprecate a Feature Group.